lib/stat_tracker.rb

```diff
@@ -89,10 +89,10 @@ def average_goals_per_game
   def total_goals_per_season
     goals_per_season = Hash.new
     @games.each do |game|
-      if goals_per_season.include?(game[:season])
```

Comment:
If you take the above comment you can change the .each block to read: `goals_per_season[game[:season]] += game[:away_goals].to_f + game[:home_goals].to_f`

No need for line 92-94 or line 96